.github/workflows/main.yml

```diff
@@ -0,0 +1,62 @@
+name: ci build
+
+on: 
+  pull_request:
+    branches: 
+      - master
+  
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    if: ${{github_head_ref}} == 'development'
+    
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - name: Setup Node 10.x
+        uses: actions/setup-node@v2-beta
+        with:
+          node-version: '10'
+      
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-node-modules
+        with:
+          # npm cache files are stored in `~/.npm` on Linux/macOS
+          path: ~/.npm
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+          
+      - name: Installing Dependencies
+        run: npm install
+     
+      - name: Patching dependencies
+        run: |
+          cat ./node_modules/@angular-devkit/build-angular/src/angular-cli-files/models/webpack-configs/browser.js
+          sed -i "s/node: false/node: {crypto: true, stream: true, fs: 'empty', net: 'empty', tls: 'empty'}/g" ./node_modules/@angular-devkit/build-angular/src/angular-cli-files/models/webpack-configs/browser.js 
+          cat ./node_modules/@angular-devkit/build-angular/src/angular-cli-files/models/webpack-configs/browser.js
+      
+      - name: build for testnet
```

Comment:
Is this just a staging environment for the Testnet build?

Reply:
yes.